File: realworld-back/realworld-common/src/main/java/com/realworld/context/BaseContext.java
package com.realworld.context;

public class BaseContext {
	private final static ThreadLocal<Integer> THREAD_LOCAL = new ThreadLocal<>();
	public static void setCurrentId(Integer id){
		THREAD_LOCAL.set(id);
	}
	public static Integer getCurrentId(){
//		return THREAD_LOCAL.get();
		return 1;
	}
	public static void removeCurrentId(){
		THREAD_LOCAL.remove();
	}
}

File: backend-springboot/src/main/kotlin/com/yyjy/common/BaseContext.kt
package com.yyjy.common

/**
 * @author Nyxcirea
 * @date 2026/2/14
 * @description: TODO
 */
class BaseContext {
    companion object {
        private val threadLocal = ThreadLocal<Int>()

        fun setCurrentId(id: Int) {
            threadLocal.set(id)
        }
        fun getCurrentId(): Int {
            return threadLocal.get()
        }
        fun removeCurrentId() {
            threadLocal.remove()
        }
    }
}